ledger-tool/src/main.rs

```diff
@@ -1413,7 +1414,7 @@ fn main() {
             )
         ).subcommand(
             SubCommand::with_name("accounts")
-            .about("Print account contents after processing in the ledger")
+            .about("Print account stats and contents after processing the ledger")
```

Comment:
As an aside (don't expect you to act on it), it seems like `cargo fmt` isn't fixing up this App (normally these modifiers would be indented). Something in the realm of this issue: https://github.com/rust-lang/rustfmt/issues/3863#issuecomment-699768919 
Note to self to either fix up long strings, or hand format at some point.